lib/fs/basicfs.go

```diff
@@ -28,6 +28,9 @@ type BasicFilesystem struct {
 	root string
 }
 
+func NewBasicFilesystem(root string) *BasicFilesystem {
```

Comment:
This should just be a rename of the existing constructor if it needs to be exported